lib/facter/postgres_version.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Facter.add('postgres_version') do
+  confine { Facter::Core::Execution.which('postgres') }
+  setcode do
+    version = Facter::Core::Execution.execute('postgres -V 2>/dev/null')
+    version.match(%r{\d+\.\d+$})[0] if version
```

Comment:
This regex won't match the output of Debian family packages:

```console
kenyon@lithium ~ % /usr/lib/postgresql/14/bin/postgres -V
postgres (PostgreSQL) 14.13 (Debian 14.13-1.pgdg110+1)
```

Reply:
Okay. Guess the regex will have to be adjusted. This is RHEL 9 by the way, and I believe also vanilla PostgreSQL:

```
% /bin/postgres -V
postgres (PostgreSQL) 13.14
%
```